### DISCUSSION TITLE
⭐ star exchange

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Schweinepriester/github-profile-achievements#770] [General]
star my repo - https://github.com/Sriansh-raj/GitHub-Achievements

and reply with your repo link for me to star

happy GitHub :octocat: 

❤️

**Comment by Schweinepriester** [comment]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722

**Reply by Sriansh-raj** [reply]
Sorry! my bad 😿
